internal/logger.go

```diff
@@ -111,6 +118,67 @@ func (ls *loggerService) StartLoggers() error {
 	return nil
 }
 
+func (ls *loggerService) PIDLoggers(vin string) error {
```

Comment:
should this method be renamed more to something like `SavePIDLoggersFromConfig(vin string)`